src/vm/excep.cpp

```diff
@@ -5234,30 +5234,31 @@ LONG InternalUnhandledExceptionFilter_Worker(
 #endif // DEBUGGING_SUPPORTED
 
 
-#if defined(FEATURE_EVENT_TRACE) && !defined(FEATURE_PAL)
-        DoReportForUnhandledException(pParam->pExceptionInfo);
-#endif // FEATURE_EVENT_TRACE    
-
         //
         // Except for notifying debugger, ignore exception if unmanaged, or
         // if it's a debugger-generated exception or user breakpoint exception.
         //
         if (tore.GetType() == TypeOfReportedError::NativeThreadUnhandledException)
         {
             pParam->retval = EXCEPTION_CONTINUE_SEARCH;
+#if defined(FEATURE_EVENT_TRACE) && !defined(FEATURE_PAL)
+            DoReportForIgnoredUnhandledException(pParam->pExceptionInfo);
+#endif
             goto lDone;
         }
 
         if (pParam->fIgnore)
         {
             LOG((LF_EH, LL_INFO100, "InternalUnhandledExceptionFilter_Worker, ignoring the exception\n"));
             pParam->retval = EXCEPTION_CONTINUE_SEARCH;
+#if defined(FEATURE_EVENT_TRACE) && !defined(FEATURE_PAL)
+            DoReportForIgnoredUnhandledException(pParam->pExceptionInfo);
+#endif
             goto lDone;
         }
 
         LOG((LF_EH, LL_INFO100, "InternalUnhandledExceptionFilter_Worker: Calling DefaultCatchHandler\n"));
 
-
         // Call our default catch handler to do the managed unhandled exception work.
         DefaultCatchHandler(pParam->pExceptionInfo, NULL, useLastThrownObject,
             TRUE /*isTerminating*/, FALSE /*isThreadBaseFIlter*/, FALSE /*sendAppDomainEvents*/);
```

Comment:
DefaultCatchHandler() now has sendWindowsEventLog=FALSE by default so it would need to be overridden here. I assume this isn't working as-is.